pr_review_list.rb

```diff
@@ -39,6 +40,7 @@ def does_array_have_pr(array, pr_number)
   # loop through open pr's and create a row that has all the pertinant info
   pr_information_cache.each do |pr|
     sleep(2)
+    if pr[:pull][:draft] == false
```

Comment:
The indentation seems off here and makes it harder to read - anything that is executed if this conditional is true, should be indented underneath:
```ruby
if something == value
  // rest of code
end  
```

Reply:
A more idiomatic Ruby implementation of this would be:
```suggestion
    unless pr[:pull][:draft]
```

More of an FYI for future

Reply:
Should I leave the way this is because if I click on resolve conversation, than my changes would no longer be there?

Reply:
Refer above :- @sanfrancrisko 

git checkout HEAD^ -- ocktokit_utils.rb
git checkout HEAD^ -- report30.html
# etc...
git add ocktokit_utils.rb
git add report30.html
# etc...

If I make this change, I will get conflicts, because when I initially raised this PR - this was were the conflicts raised, that's why I **accepted the suggested change by VS Code Editor** Please let me know if the two changes requested still needs to be done and how?

Reply:
Can you provide more clarity on the changes suggested? I am just scared if the same conflicts will raise again, as it happened the very first time when this pr was raised, thank you @sanfrancrisko

Reply:
You can choose to use the new `unless` method or keep the same - I'll leave that decision with you. Ultimately though, the main issue is the indentation issues that have been raised by myself, @tphoney and @pmcmaw.

Resolution of conversations will be automatically done by Github if you push a code change that address the issue on the line the comments are on. If not, you can manually resolve after.

Reply:
Regarding conflicts, there are already conflicts being reported by Github on the PR anyway, as you can see below, so you will need to resolve this regardless. 

Have you tried the steps outlined in my comment here:

> There appears to be a number of formatting changes, possibly caused by the IDE, that have been checked in as part of this change (e.g. in `report30.html`, `report60.html`, `report90.html`, `octokit_utils.rb`) - if these are not related to the change (_and as far as I can see, they are not_), then do not include them in the commit.
> 
> I would recommend reverting those file changes when addressing the other comments on the review:
> 
> ```
> git checkout HEAD^ -- ocktokit_utils.rb
> git checkout HEAD^ -- report30.html
> # etc...
> ```
> 
> ```
> git add ocktokit_utils.rb
> git add report30.html
> # etc...
> ```

The idea here is that you simply use the version of those files that were on branch before you made any edits. If there's still conflicts, then it would suggest that the branch was not created from an up-to-date version of `puppetlabs:main`, in which case I would recommend performing a rebase.

Reply:
Not yet,  so I should run these commands in the terminal with the update version of the community_management repo?

Reply:
Those commands should be run at the project root of `community_management`. I'm not sure what you mean by:
> with the update version of the community_management repo
Can you please outline what you mean by this?

What might be better is just pulling in the version of those files from `puppetlabs:main`. Assuming you have configured this as a remote repo named `upstream`:
```
git checkout upstream/main -- ocktokit_utils.rb
```

Given you are targeting this PR to `puppetlabs:main`, this should resolve conflicts, because you now have an identical version of the file(s) on `Disha-maker:IAC-1196/main/Exclude_drafts_from_module_report` and `puppetlabs:main`

Reply:
This is what I mean.... 
I will have to clone down the last version of the community management repo so that they are identical, the next time I will work on it, right?